samples/meetings-stage-view/csharp/README.md

```diff
@@ -171,3 +171,4 @@ For reference please check [Enable and configure your apps for Teams meetings](h
 ## Further Reading.
 
 [Meeting stage view](https://learn.microsoft.com/en-us/microsoftteams/platform/sbs-meetings-stage-view)
```

Comment:
We need to update the README for app id/external id for deep link to work locally.

Reply:
Updated readme